### PR TITLE
feature: add endpoint to update user profile emp_id with perdet_seq_num from fsbid

### PR DESCRIPTION
**Pull request by rtirserio**
This will be called on login from the front end to set this value which is necessary for bidding

### CODE REVIEW

**Comment by mjoyce91** [comment]
Getting a JSONDecodeError when this is called from the front-end using both a JWT that I added a `unique_name` property to, as well as the fake JWT from the test. I've also updated my mock-fsbid repo and rebuilt. Anything else I need to do to test this?

**Comment by rtirserio** [comment]
@mjoyce91 i dont think so. you may want to try to call the authorize mock endpoint with the tm_usrname header to one of the usernames in the mock

**Comment by mjoyce91** [comment]
Ah thanks, I was using a fake username

File: talentmap_api/fsbid/views/employee.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class FSBidEmployeePerdetSeqNumActionView(BaseView):
+
+    permission_classes = (IsAuthenticated,)
+
+    def put(self, request, *args, **kwargs):
```
Comment (mjoyce91) [inline talentmap_api/fsbid/views/employee.py]: Should this work if the `emp_id` has already been set? It works for me the first time, but subsequent calls don't update the user's `emp_id`.

Comment (rtirserio) [reply talentmap_api/fsbid/views/employee.py]: Its not something that should change but it should work each time the user logs in

Comment (mjoyce91) [reply talentmap_api/fsbid/views/employee.py]: Okay I see what I was doing wrong. Looks good, approving

**Pull request merged**